zips/zip-0000.rst

```diff
@@ -410,13 +432,50 @@ Each ZIP SHOULD have the following parts:
 * Terminology — Definitions of technical or non-obvious terms used
   in the document.
 
-* Abstract — A short (~200 word) description of the technical issue
-  being addressed.
+* Abstract — The abstract SHOULD be a concise summary (approximately
+  200 words) that clearly describes the issue the ZIP addresses, the
+  proposed solution, and the intended outcome. The ZIP MUST remain
+  complete without the abstract.
+
+  The abstract SHOULD, as far as practical, provide enough context for
+  readers to quickly understand what the ZIP is about, why it matters,
+  and how it fits into the broader Zcash ecosystem. In addition to
+  summarizing the technical content, the abstract SHOULD briefly mention
+  any significant privacy implications. The abstract should not be
+  vague or overly generic, but should serve as a faithful, high-level
+  summary of the proposal that enables reviewers to understand the key
+  ideas and potential implications without reading the entire document.
 
 * Motivation — The motivation is critical for ZIPs that want to change
   the Zcash protocol. It should clearly explain why the existing
   protocol is inadequate to address the problem that the ZIP solves.
 
+* Privacy Implications — This section SHOULD be present if the ZIP makes
+  trade-offs or assumptions that could impact the privacy of Zcash users
+  or protocol participants. Since this section is not part of the
+  Specification, it MUST NOT directly include or duplicate conformance
+  requirements on implementations. Instead it contains a high-level overview
+  of the privacy implications that may need to be considered for deployment
+  of the ZIP.
+
+  The contents of this section could include, for example, trust assumptions
+  affecting privacy that are introduced or modified by the proposal; privacy
+  trade-offs or changes to anonymity guarantees; potential data leakage
+  (e.g., from metadata, timing, or usage patterns); and exposure to known
+  or novel privacy attack vectors. Proposals SHOULD consider any relevant
+  threat model(s), and SHOULD propose updates to those threat models if the
+  ZIP introduces new privacy-relevant assumptions or risks not currently
+  addressed.
+
+  If a change affects how wallets construct, interpret, or expose
+  transaction data, or modifies user-facing protocol behavior, the ZIP
+  SHOULD assess the implications for user privacy in that context.
+
+* Requirements — This section describes high-level goals that the
+  Specification MUST ensure. It MUST NOT include conformance requirements,
+  so that it is possible to perform a consistency check that what is
+  specified meets the Requirements.
```

Comment:
```suggestion
  Specification MUST ensure. It MUST NOT directly include conformance
  requirements, so that it is possible to perform a consistency check that
  what is specified meets the Requirements.
```